response_operations_ui/templates/reporting-unit-survey.html

```diff
@@ -0,0 +1,257 @@
+{% extends "layouts/base.html" %}
+{% from "components/button/_macro.njk" import onsButton %}
+{% from "components/table/_macro.njk" import onsTable %}
+
+{% set page_title = ru.name + " | " + survey[1] %}
+
+{% block main %}
+    <div class="grid">
+        <div id="main" class="grid__col col-10@m">
+            <form action="{{ url_for('reporting_unit_bp.view_reporting_unit', ru_ref=ru.sampleUnitRef) }}" method="get">
```

Comment:
I know it was part of the design but looking at it, it's a bit weird.  Also the csrf_token is in the url which I'm not sure is right either.

If this always goes back to the view_reporting_unit page `/reporting-units/<an ru_ref>` then why can't this just be a link instead of a post?